app/kubernetes/views/applications/create/createApplicationController.js

```diff
@@ -793,13 +789,8 @@ class KubernetesCreateApplicationController {
     return this.formValues.PublishingType === KubernetesApplicationPublishingTypes.LOAD_BALANCER && _.filter(this.formValues.PublishedPorts, { IsNew: false }).length;
   }
 
-  isProtocolOptionDisabled(index, protocol) {
-    return (
-      this.disableLoadBalancerEdit() ||
-      (this.isEditAndNotNewPublishedPort(index) && this.formValues.PublishedPorts[index].Protocol !== protocol) ||
-      (this.isEditLBWithPorts() && this.formValues.PublishedPorts[index].Protocol !== protocol) ||
-      (this.isNewAndNotFirst(index) && this.formValues.PublishedPorts[index].Protocol !== protocol)
-    );
+  isProtocolOptionDisabled(index) {
```

Comment:
You still need the protocol it seems.

This function is called by:
- `createApplication.html`
  - line 1527: `ng-disabled="ctrl.isProtocolOptionDisabled($index, 'TCP')"`
  - line 1535: `ng-disabled="ctrl.isProtocolOptionDisabled($index, 'UDP')"`

Reply:
please ignore the previous changes, now issue fixed in the correct way.